src/gameplay/game.ts

```diff
@@ -1988,6 +1995,41 @@ class Game extends Room {
     );
   }
 
+  voteVoidGame(socket: SocketUser): void {
+    // Verify they are in the game.
+    if (!this.usernameIsPlayer(socket.request.user.username)) {
+      socket.emit('messageCommandReturnStr', {
+        message: 'You are not a player in this game.',
+        classStr: 'server-text',
+      });
+      return;
+    }
+
+    const numResPlayers = this.playersInGame.filter(
+      (player) => player.alliance === Alliance.Resistance,
+    ).length;
+    const votesNeeded = numResPlayers + 1;
+    const numVoted = this.voidGame.voteVoidGame(
+      socket.request.user.username,
+      votesNeeded,
+    );
+
+    const s = numVoted > 1 ? 's have' : ' has';
+
+    this.sendText(
+      `${numVoted} player${s} voted to void the game. ${votesNeeded} votes needed.`,
+      'server-text',
+    );
+
+    if (numVoted >= votesNeeded) {
```

Comment:
`if (numVoted >= votesNeeded) ` lets move this logic into the voidGameVoteTracker